python_modules/libraries/dagster-dbt/dagster_dbt/cli/resources_v2.py

```diff
@@ -0,0 +1,132 @@
+import json
+import os
+import subprocess
+from dataclasses import dataclass
+from typing import Any, Dict, Iterator, List, Sequence
+
+from dagster import _check as check
+
+
+@dataclass
+class DbtCliEventV2:
```

Comment:
I worry that this class is not long for this world, as once we start interfacing with the dbt python API, we'll likely just want to yield raw EventMsg objects from dbt execution.

Is there a way we can avoid having three objects to represent "dbt message"? (those would be `DbtCliEvent`, `DbtCliEvent2`, and `EventMsg`)

Reply:
I suppose we could just rename the current `DbtCliEvent` (which we plan to deprecate) `DeprecatedDbtCliEvent` or something, as it's not part of the public interface